instrumentation/jms-1.1/src/latestDepTest/groovy/JMS2Test.groovy

```diff
@@ -180,7 +180,7 @@ class JMS2Test extends AgentTestRunner {
       trace(0, 1) { // Consumer trace
         span(0) {
           parent()
-          operationName "jms.receiveNoWait"
+          operationName expectedSpanName
```

Comment:
I'm amazed this is how the test is updated :P

But I'm not sure this is right - the span name is supposed to be an operation so a verb makes sense to me. Instead of `jms.receiveNoWait` and `jms.receive`, I'd expect them both to be just `receive`.

The queue or topic instead goes in `messaging.destination`

https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/messaging.md#messaging-attributes

Reply:
from that link:

> Span name: The span name should usually be set to the message destination name

Reply:
Ah thanks, missed that

Reply:
By the way filed https://github.com/open-telemetry/opentelemetry-specification/issues/686 if that goes somewhere I'll take care of updating :)